prow/scripts/cluster-integration/compass-gke-integration.sh

```diff
@@ -0,0 +1,360 @@
+    --data "global.alertTools.credentials.slack.channel=${KYMA_ALERTS_CHANNEL}" \
+    --data "global.alertTools.credentials.slack.apiurl=${KYMA_ALERTS_SLACK_API_URL}" \
+    --data "pushgateway.enabled=true" \
+    --label "component=monitoring"
+
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --name "istio-overrides" \
+    --data "gateways.istio-ingressgateway.loadBalancerIP=${GATEWAY_IP_ADDRESS}" \
+    --label "component=istio"
+}
+
+function applyCompassOverrides() {
+  NAMESPACE="compass-installer"
+
+  if [ "${RUN_PROVISIONER_TESTS}" == "true" ]; then
+    # Change timeout for kyma test to 3h
+    export KYMA_TEST_TIMEOUT=3h
+
+    # Create Config map for Provisioner Tests
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --namespace "${NAMESPACE}" --name "provisioner-tests-overrides" \
+      --data "global.provisioning.enabled=true" \
+      --data "provisioner.security.skipTLSCertificateVeryfication=true" \
+      --data "provisioner.tests.enabled=true" \
+      --data "provisioner.gardener.kubeconfig=$(base64 -w 0 < "${GARDENER_APPLICATION_CREDENTIALS}")" \
+      --data "provisioner.gardener.project=$GARDENER_PROJECT_NAME" \
+      --data "provisioner.tests.gardener.azureSecret=$GARDENER_AZURE_SECRET_NAME" \
+      --label "component=compass"
+  fi
+
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --namespace "${NAMESPACE}" --name "compass-auditlog-mock-tests" \
+    --data "global.externalServicesMock.enabled=true" \
+    --data "gateway.gateway.auditlog.enabled=true" \
+    --data "gateway.gateway.auditlog.authMode=oauth" \
+    --label "component=compass"
+}
+
+function applyCommonOverrides() {
+  NAMESPACE=${1}
+
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --namespace "${NAMESPACE}" --name "installation-config-overrides" \
+    --data "global.domainName=${DOMAIN}" \
+    --data "global.loadBalancerIP=${GATEWAY_IP_ADDRESS}"
+
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --namespace "${NAMESPACE}" --name "feature-flags-overrides" \
+    --data "global.enableAPIPackages=true" \
+    --data "global.disableLegacyConnectivity=true"
+
+
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --namespace "${NAMESPACE}" --name "cluster-certificate-overrides" \
+    --data "global.tlsCrt=${TLS_CERT}" \
+    --data "global.tlsKey=${TLS_KEY}"
+}
+
+function installKyma() {
+  kubectl create namespace "kyma-installer"
+  applyCommonOverrides "kyma-installer"
+  applyKymaOverrides
+
+  echo "Use Kyma artifacts from KYMA_VERSION file"
+  readonly KYMA_VERSION=$(cat "${COMPASS_RESOURCES_DIR}/KYMA_VERSION")
+  readonly KYMA_ARTIFACTS="${KYMA_DEVELOPMENT_ARTIFACTS_BUCKET}/${KYMA_VERSION}"
+  readonly TMP_DIR="/tmp/compass-gke-integration"
+  gsutil cp "${KYMA_ARTIFACTS}/kyma-installer-cluster-compass-dependencies.yaml" ${TMP_DIR}/kyma-installer-cluster-compass-dependencies.yaml
```

Comment:
We shouldn't depend on a file on Kyma repo.
Please create a PR to compass repo with file with Kyma components in `installation/resources` directory and use it here.

Reply:
Agree, however this file is already copied to bucket (as it was need for the PoC) so we can use it before we introduce changes to our pipeline and generate our copies of kyma installer combo yamls.

The contents of the file won't change, so I think we can replace this in the PR with changes to artifacts generation.

Reply:
The changes were done in this PR.